src/shared/views/Login.js

```diff
@@ -474,24 +482,24 @@ class _Login extends React.Component<Props, State> {
                         }
                     />
                     <Text style={styles.checkboxLabel}>
-                        I agree to the&nbsp;
-                        <Text
-                            style={styles.policyLink}
-                            onPress={() => {
-                                navigation.push('WebviewWindow', {
-                                    uri: 'https://mapswipe.org/privacy',
-                                });
-                            }}
-                        >
-                            Privacy Notice
-                        </Text>
+                        <Trans i18nKey="signup:IagreeToPrivacyNotice">
+                            I agree to the bal
```

Comment:
`bal` ?

Reply:
Not sure where `bal` came from, in practice it doesn't matter, since it's overwritten by the translated strings at runtime. I've removed it to make it clearer :)